css/index.css

```diff
@@ -0,0 +1,41 @@
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
+
+body {
+    background-color: rgb(234, 255, 0);
+    color: grey;
+    font-family: Arial, Helvetica, sans-serif;
+    text-align: center;
+}
+
+img {
+    border-radius: 75px;
+    border: red 5px solid;
+}
+
+h1,
+.connecting {
+    font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+}
+
+h2 {
+    font-size: 200%;
+    font-weight: bold;
+    color: red;
+    border: blue 3px solid;
+    padding: 10px;
+    margin: 10px 20px;
+}
+
+#experience {
```

Comment:
Look at border section in scrimba - I think AI reviewer wanted a border with color/shadows, etc.